force-app/main/default/classes/PS_GatewayManagement.cls

```diff
@@ -43,23 +43,58 @@ public with sharing class PS_GatewayManagement {
         };
         PS_IntegrationService.setConfiguration(gatewayConfig);
     }
-
     @AuraEnabled
+    public static void setGatewayAssignmentEnabled(Boolean gatewayAssignmentEnabled) {
+        Gift_Entry_Settings__c giftEntrySettings = UTIL_CustomSettingsFacade.getGiftEntrySettings();
+        giftEntrySettings.Enable_Gateway_Assignment__c = gatewayAssignmentEnabled;
+
+        upsert giftEntrySettings;       
+    }
+
+    public static Boolean isGatewayAssignmentEnabled() {
```

Comment:
This method can now be `private` and annotated with `@TestVisible` so its available to be called in your unit tests